app/src/main/java/com/gholem/moneylab/features/add/adapter/viewholder/AddTransactionViewHolder.kt

```diff
@@ -32,7 +33,8 @@ sealed class AddTransactionViewHolder(binding: ViewBinding) :
             binding.amountEditText.setText(transaction.amount)
 
             if (isInvalidData) {
-                binding.amountInputLayout.error = "Value cannot be empty"
+                binding.amountInputLayout.error =
+                    Resources.getSystem().getString(R.string.error_message)
```

Comment:
Never use resources in that way. You have to do that from context/view, so please use this instead:

`val resources = binding.root.resources`